pro/src/screens/OfferEducationalStock/types.ts

```diff
@@ -0,0 +1,12 @@
+export type OfferEducationalStockFormValues = {
+  eventDate: Date
+  eventTime: string
+  numberOfPlaces: number | ''
+  totalPrice: number | ''
+  bookingLimitDatetime: Date
+}
+
+export type Offer = {
+  id: string
+  status: string
```

Comment:
désolé pour la validation sans réserve : 

```suggestion
  status: OfferStatus
```

et autre part 

```ts
type OfferStatus =  'ACTIVE' | 'INACTIVE' | 'SOLD_OUT' | 'EXPIRED' | 'PENDING' | 'REJECTED' | 'DRAFT'
```

et aussi : 

```ts
export const OFFER_STATUS_ACTIVE: OfferStatus = 'ACTIVE'
 export const OFFER_STATUS_INACTIVE: OfferStatus = 'INACTIVE'
 export const OFFER_STATUS_SOLD_OUT: OfferStatus = 'SOLD_OUT'
 export const OFFER_STATUS_EXPIRED: OfferStatus = 'EXPIRED'
 export const OFFER_STATUS_PENDING: OfferStatus = 'PENDING'
 export const OFFER_STATUS_REJECTED: OfferStatus = 'REJECTED'
 export const OFFER_STATUS_DRAFT: OfferStatus = 'DRAFT'
```

Reply:
tant qu'à faire un enum non ? 😇😇😇😇😇😇😇😇😇 ca fait les 2 à la fois

Reply:
J'avoue, j'ai été mauvais

Reply:
@MathildeDuboille tu es sensée prendre ma défense!!! Pas me rajouter du taf!! 😭